xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -90,8 +98,16 @@
         <param name="consumed">Marks the extent of the data that has been successfully processed.</param>
         <param name="examined">Marks the extent of the data that has been read and examined.</param>
         <summary>Moves forward the pipeline's read cursor to after the consumed data.</summary>
-        <remarks>The memory for the consumed data will be released and no longer available.
-            The examined data communicates to the pipeline when it should signal more data is available.
+       <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```

Comment:
When all overloads have the same remarks, I prefer creating a Member Group to have a single remarks section. but it's a nit.

Reply:
Done. Did I do it right? Can you please take another look?